README.md

```diff
@@ -20,17 +20,25 @@ To use the postgresql in your project have your tests depend on starting the Pos
 ```
 lazy val root = (project in file(".")).enablePlugins(EmbeddedPostgresPlugin)
 
-testOptions in Test <+= postgresTestCleanup // clean up postgresql after tests
+testOptions in Test += postgresTestCleanup.value // clean up postgresql after tests
 
-test in Test <<= (test in Test).dependsOn(startPostgres)
+test in Test := (test in Test).dependsOn(startPostgres)
 
-testOnly in Test <<= (testOnly in Test).dependsOn(startPostgres)
+testOnly in Test := (testOnly in Test).dependsOn(startPostgres)
```

Comment:
When run with `sbt.version=0.13.11` the above `build.sbt` config changes didn't work right. Running `sbt test` just produces following output without any tests
```
[info] Loading global plugins from /Users/foo/.sbt/0.13/plugins
[info] Loading project definition from /Users/foo/path/to/my-project/project
[info] Set current project to my-project (in build file:/Users/foo/path/to/my-project)
[success] Total time: 0 s, completed 09-Oct-2017 15:44:22
```

Reply:
The configs in question:

```
// these worked when I tried them:
// testOptions in Test <+= postgresTestCleanup
// test in Test <<= (test in Test).dependsOn(startPostgres)
// testOnly in Test <<= (testOnly in Test).dependsOn(startPostgres)

// these changes didn't run for me:
testOptions in Test += postgresTestCleanup.value
test in Test := (test in Test).dependsOn(startPostgres)
testOnly in Test := (testOnly in Test).dependsOn(startPostgres)
```

Reply:
Yeah, you're right. The documentation is intended to be used with `sbt.version=0.13.16`.

How do you want me to address this issue? I could:
* revert the documentation to be used with your sbt-version and ignore the deprecation warnings with higher sbt versions.
* add documentation for old and new sbt versions.

Reply:
I'm happy with either of the 2 suggestions, so either one that suits you better :)